constants/permission-scope-constants.js

```diff
@@ -0,0 +1,8 @@
+module.exports = Object.freeze({
+    ALL: "all",
+    NONE: "none",
+    OWN: "own",
+    STUDY: "study",
+    DC: "dc",
```

Comment:
I think the value should also be "DC", since it's an acronym. Peter is using "DC" in the PBAC JSON file also.